addons/account_peppol/models/res_company.py

```diff
@@ -453,3 +490,47 @@ def _account_peppol_send_welcome_email(self):
             return
 
         mail_template.send_mail(self.id, force_send=True)
+
+    def _peppol_get_default_feature_flags(self):
+        self.ensure_one()
+        return {
+            'rate_limiting_ui': True,
+            'proactive_peppol_prompt_countries': {
+                'country_codes': [
+                    'BE', 'FI', 'LU', 'LV', 'NL', 'NO', 'SE',
```

Comment:
Extract in a global const since it's in common with `info_always_on_countries` now.